src/ActionList/List.tsx

```diff
@@ -36,12 +43,17 @@ interface ListPropsBase {
    * - `"full"` - `List` children are flush (vertically and horizontally) with `List` edges
    */
   variant?: 'inset' | 'full'
+
+  // **
+  tabIndex?: number
+
+  ref?: React.RefObject<HTMLDivElement>
```

Comment:
is there some sort of predefined type or interface we can use for these?

Reply:
Both `tabIndex` and `ref` would be available if the `ListPropsBase` interface `extends`’d `React.ComponentPropsWithRef<'div'>`